fvm/evm/testutils/contracts/test.sol

```diff
@@ -48,6 +48,13 @@ contract Storage {
         selfdestruct(payable(msg.sender));
     }
 
+    function verifyArchCallToRandomSource(uint64 height) public view returns (uint64) {
+        (bool ok, bytes memory data) = cadenceArch.staticcall(abi.encodeWithSignature("getRandomSource(uint64)", height));
+        //require(ok, "unsuccessful call to arch ");
```

Comment:
Should this check be commented?

Reply:
nope :D

Reply:
it's not in the binary, just forgot to update the source